packages/frontend/src/redux/createReducers/index.js

```diff
@@ -0,0 +1,9 @@
+import { combineReducers } from 'redux';
+
+import combinedMainReducers from './combinedMainReducers';
+import setupAccountReducer from './setupAccountReducer';
+
+export default (history, accountId) => combineReducers({
+    ...combinedMainReducers(history),
+    accountsReducer: combineReducers(setupAccountReducer(accountId))
```

Comment:
Let's just call this `accounts`. `accountsReducer` is redundant since all keys in the object passed to `combineReducers` have values which are reducers.

Really we just need something that doesn't conflict with `allAccounts` and makes it clear that we have account data under it.

Reply:
Sure. Done.